src/common/components/hosts/HostStatus.tsx

```diff
@@ -138,6 +152,15 @@ const HostStatusPopoverContent: React.FC<HostStatusPopoverContentProps> = (props
           {statusDetails}
           <br />
           {toSentence(statusInfo)}
+          <br />
+          {hints
+            .filter((hint) => hint)
```

Comment:
What is this line filtering exacly?

Reply:
Empty strings so that we don't get a bunch of empty lines in case there are multiple failed validations.